wiki/content/slash-graphql/slash-quick-start.md

```diff
@@ -85,11 +109,98 @@ Let's also query back the users and their tasks
 }
 ```
 
-You'll see that Slash figured out that users are unique by their username, and so you only see a single record for each user.
+As in the previous step, to execute the query, paste the code into the API Explorer, and hit the **Execute Query** button.
+
+The query's results are shown below. If you look carefully, you'll see that Slash figured out that users are unique \(by the `username`\), and it has returned a single record for each user.
+
+```graphql
+{
+  "data": {
+    "queryUser": [
+      {
+        "username": "skywalker@dgraph.io",
+        "tasks": [
+          {
+            "title": "Dismantle Empire"
+          },
+          {
+            "title": "Meet Dad"
+          }
+        ]
+      },
+      {
+        "username": "your-email@example.com",
+        "tasks": [
+          {
+            "title": "Write A Schema"
+          },
+          {
+            "title": "Profit!"
+          },
+          {
+            "title": "Create a database"
+          },
+          {
+            "title": "Put Data In"
+          },
+          {
+            "title": "Complete Tasks with UI"
+          }
+        ]
+      },
+      {
+        "username": "frodo@dgraph.io",
+        "tasks": [
+          {
+            "title": "More Walking"
+          },
+          {
+            "title": "Discard Jewelery"
+          },
+          {
+            "title": "Walking"
+          }
+        ]
+      }
+    ]
+  },
+  "extensions": {
+    "touched_uids": 40,
+    "tracing": {
+      "version": 1,
+      "startTime": "2020-08-15T12:59:14.577795285Z",
+      "endTime": "2020-08-15T12:59:14.580373297Z",
+      "duration": 2578026,
+      "execution": {
+        "resolvers": [
+          {
+            "path": [
+              "queryUser"
+            ],
+            "parentType": "Query",
+            "fieldName": "queryUser",
+            "returnType": "[User]",
+            "startOffset": 79480,
+            "duration": 2422504,
+            "dgraph": [
+              {
+                "label": "query",
+                "startOffset": 125039,
+                "duration": 2291416
+              }
+            ]
+          }
+        ]
+      }
+    },
+    "queryCost": 1
+  }
+}
+```
 
-## Auth
+## Authorization
 
-Now that we have a schema working, let's update that schema to add some authorization. We'll update the schema so that users can only read their own tasks back.
+Now that we have a working schema, let's update the original code and add some access authorization. We'll update the schema so that users can only read their tasks back. \(E.g. Frodo won't be able to read Luke's ToDo tasks\)
```

Comment:
Suggested edit:

can only read tasks that they own \(for example, this change would prevent Frodo from reading Luke's tasks\).

Reply:
fixed